var/spack/repos/builtin/packages/namd/package.py

```diff
@@ -24,17 +24,14 @@ class Namd(MakefilePackage, CudaPackage):
     maintainers("jcphill")
 
     version("master", branch="master")
+    version("3.0b3", sha256="20c32b6161f9c376536e3cb97c3bfe5367e1baaaace3c716ff79831fc2eb8199")
     version("2.15a2", sha256="8748cbaa93fc480f92fc263d9323e55bce6623fc693dbfd4a40f59b92669713e")
     version("2.15a1", branch="master", tag="release-2-15-alpha-1")
     # Same as above, but lets you use a local file instead of git
     version(
         "2.15a1.manual", sha256="474006e98e32dddae59616b3b75f13a2bb149deaf7a0d617ce7fb9fd5a56a33a"
     )
-    version(
-        "2.14",
-        sha256="34044d85d9b4ae61650ccdba5cda4794088c3a9075932392dd0752ef8c049235",
-        preferred=True,
```

Comment:
Does removing preferred=True make 3.0b3 the default install? I wouldn't make a beta release the default.

Reply:
Good catch! 

> I wouldn't make a beta release the default.

Totally agre. This is something I removed during my tries with `3.0b3` and forgot to revert it.